notifications_utils/field.py

```diff
@@ -63,8 +63,8 @@ class Field:
     """
 
     placeholder_pattern = re.compile(
-        r'\({2}'        # opening ((
-        r'([^()]+)'     # body of placeholder - potentially standard or conditional.
+        r'\({2}(?=[^(])'        # opening ((
```

Comment:
It took me a while to understand what this is doing: we need to account for three opening brackets. Can we extend the comment to explain?